tests/js/common/shell/shell-pregel-components.js

```diff
@@ -435,4 +439,5 @@ jsunity.run(pagerankTestSuite);
 jsunity.run(seededPagerankTestSuite);
 jsunity.run(ssspTestSuite);
 jsunity.run(hitsTestSuite);
+// jsunity.run(effectiveClosenessTestSuite);
```

Comment:
Is this commented out because effective closeness is broken? then at least this needs a comment; otherwise, delete